lib/ansible/module_utils/network/netconf/netconf.py

```diff
@@ -18,13 +18,50 @@
 #
 import json
 
+from copy import deepcopy
 from contextlib import contextmanager
 
-from ansible.module_utils._text import to_text
+try:
+    from lxml.etree import fromstring, tostring
+except ImportError:
+    from xml.etree.ElementTree import fromstring, tostring
+
+from ansible.module_utils._text import to_text, to_bytes
+from ansible.module_utils.basic import env_fallback
 from ansible.module_utils.connection import Connection, ConnectionError
 from ansible.module_utils.network.common.netconf import NetconfConnection
 
 
+netconf_provider_spec = {
```

Comment:
As far as I can tell, `netconf_config` did not previously have a `provider` key. I don't necessarily think it is problematic to add `provider` here, I just don't really see the point of making `connection=local` any easier to use when we would rather people move to `connection=netconf` instead.